src/pages/api/auth/me.ts

```diff
@@ -1,22 +0,0 @@
-import {
```

Comment:
We  no longer ‘hack’ the currently logged in user to avoid typecasting default auth0 types. This simplifies the codebase. We can revise the approach once we’ve finished Next.js and start working on new features.